zerver/openapi/zulip.yaml

```diff
@@ -1910,6 +1910,36 @@ paths:
                         "msg": "User not authorized for this query",
                         "result": "error"
                     }
+  /streams/{stream_id}:
+    delete:
+      description: Delete the stream.
+      parameters:
+      - name: stream_id
+        in: path
+        description: Delete the particular stream.
+        schema:
+          type: integer
+          required: true
+      security:
+      - basicAuth: []
+      responses:
+        '200':
+          description: Success.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/JsonSuccess'
+        '400':
+          description: Bad request.
+          content:
+            application/json:
+              schema:
+                - $ref: '#/components/schemas/JsonError'
+                - example:
+                    {
+                        "msg": "Unable to delete",
+                        "result": "error"
+                    }
```

Comment:
Did you get this example error payload by generating an actual error?

Reply:
I've updated this with the actual error we get for this endpoint.